src/runtime/assemblymanager.cs

```diff
@@ -185,7 +185,7 @@ internal class AssemblyManager {
             try {
                 assembly = Assembly.Load(name);
             }
-            catch (System.Exception e) {
+            catch (System.Exception /*e*/) {
```

Comment:
@tonyroberts according to Jon Skeet http://stackoverflow.com/a/8009801/2230844 it is possible to "miss" some unmanaged exceptions when `catch` is used without arguments.

However I have not tested this hypothesis myself, it just looked the only weird place to me.